net.sf.eclipsecs.checkstyle/metadata/com/puppycrawl/tools/checkstyle/checks/coding/checkstyle-metadata.properties

```diff
@@ -132,6 +132,9 @@ NestedTryDepth.desc = Restricts nested try blocks to a specified depth (default
 NestedTryDepth.max  = allowed nesting depth
 NestedTryDepth.name = Nested Try Depth
 
+NoArrayTrailingComma.desc = Checks that array initialization do not contain a trailing comma. Rationale: JLS allows trailing commas in arrays and enumerations, but does not allow them in other locations. To unify the coding style, the use of trailing commas should be prohibited.
```

Comment:
mix of singular and plural at "initialization do". must bei either
* initialization does
* initializations do

Reply:
this issue comes from checkstyle documentation - https://checkstyle.org/config_coding.html#NoArrayTrailingComma , 
it is not an issue at  https://checkstyle.org/checks.html (search for NoArrayTrailingComma) 

lets use description from https://checkstyle.org/checks.html .

I hope one we will find contributor to implement metadata generation based on our javadoc/xdoc files. We already did huge update to keep in sync javadoc and xdoc by UT, so metadata generation is doable.
Looks like we missed verification of Checks list page with javadoc.

Reply:
@romani Not sure what you are requesting now?
A) changing the text as I said
B) ignoring the grammar issue for the time being, so we can merge this more quickly

Reply:
I am in favor of "A)", it is easy

Reply:
I used the description from https://checkstyle.org/checks.html.

Reply:
I sorted the CustomImportOrder violation messages to match the documentation at https://checkstyle.sourceforge.io/config_imports.html#CustomImportOrder_Violation_Messages since I was adding one to the list.